gulpfile.js

```diff
@@ -66,14 +66,27 @@ gulp.task('watch-app', ['check-terriajs-dependencies'], function(done) {
     watchWebpack(webpack, webpackConfig, done);
 });
 
-gulp.task('copy-terriajs-assets', function() {
+gulp.task('symlink-terriajs-regionmapping', function() {
```

Comment:
Why symlink it rather than copy it like we do everything else? Actually, I think we're already copying it, do we just need to teach Terria to look for it in `build/TerriaJS/data` or whatever? If we need to stick with the symlink, can you confirm that this works in prod deployments too?

Reply:
Basically, because TerriaMap's wwwroot directory is both a source and a destination. So, once we copy terriajs's files into wwwroot/data, we can't tell if terriamap/wwwroot/data/regionMapping.json is a source file (not to be clobbered) or a file copied from TerriaJS (which should be clobbered).

This was kind of a lazy way of getting around the problem.

> If we need to stick with the symlink, can you confirm that this works in prod deployments too?

That's what the "dereference: true" is for later on. It worked in my superficial testing.

Reply:
>Actually, I think we're already copying it, do we just need to teach Terria to look for it in build/TerriaJS/data or whatever? 

Oh yeah, that's another approach. I did briefly try that and thought it failed, but now I see that  `/build/TerriaJS/data/regionMapping.json` loads just fine.

If we want it to be overridable with a deployment-specific version we'd need to add a bit of logic into the loading of that file.

Reply:
Could probably just add a setting to config.json telling Terria where to look for it. And then the regionids files would be resolved relative to the regionMapping.json file.